README.md

```diff
@@ -37,7 +35,7 @@ For example:
 jobs:
   build:
     docker:
-      - image: cimg/base:2020.12
+      - image: cimg/base:2022.01
```

Comment:
This is the chicken and egg problem we discussed. I wouldn't use the current image here as this is the base image. I would use an older one that is much less likely to be updated, or better yet, an image digest.

Reply:
This is the readme, ignore my comment.

Reply:
confirmed this change doesn't needed to be made